File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/GetCrossmapSetsResponse.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for getCrossmapSetsResponse complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="getCrossmapSetsResponse">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="crossmapSetResponse" type="{http://service.ws.semantikos.minsal.cl/}CrossmapSetsResponse" minOccurs="0"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "getCrossmapSetsResponse", propOrder = {
    "crossmapSetResponse"
})
public class GetCrossmapSetsResponse {

    protected CrossmapSetsResponse crossmapSetResponse;

    /**
     * Gets the value of the crossmapSetResponse property.
     * 
     * @return
     *     possible object is
     *     {@link CrossmapSetsResponse }
     *     
     */
    public CrossmapSetsResponse getCrossmapSetResponse() {
        return crossmapSetResponse;
    }

    /**
     * Sets the value of the crossmapSetResponse property.
     * 
     * @param value
     *     allowed object is
     *     {@link CrossmapSetsResponse }
     *     
     */
    public void setCrossmapSetResponse(CrossmapSetsResponse value) {
        this.crossmapSetResponse = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/getCrossmapSetsResponse.php
<?php

namespace Semantikos\ClientBundle\API;

class getCrossmapSetsResponse
{

    /**
     * @var CrossmapSetsResponse $crossmapSetResponse
     */
    protected $crossmapSetResponse = null;

    
    public function __construct()
    {
    
    }

    /**
     * @return CrossmapSetsResponse
     */
    public function getCrossmapSetResponse()
    {
      return $this->crossmapSetResponse;
    }

    /**
     * @param CrossmapSetsResponse $crossmapSetResponse
     * @return getCrossmapSetsResponse
     */
    public function setCrossmapSetResponse($crossmapSetResponse)
    {
      $this->crossmapSetResponse = $crossmapSetResponse;
      return $this;
    }

}
